clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -701,7 +701,10 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
                         memref.marker.marker_value, nullptr,
                         TESTANY(OFFLINE_FILE_TYPE_ENCODINGS, shard->file_type_),
                         /*at_kernel_event=*/true);
-                    report_if_false(shard, discontinuity.empty(), discontinuity);
+                    const std::string error_msg_suffix =
+                        "Discontinuity between instruction and kernel event marker";
```

Comment:
Let's try to abbreviate this message a bit. I'm concerned about storage costs. Maybe just add "@kernel_event marker"?